app/routes/thank-you/route.tsx

```diff
@@ -39,7 +41,5 @@ export default function ThankYouPage() {
 
 export function ErrorBoundary() {
     const error = useRouteError();
-    const title = isRouteErrorResponse(error) ? 'Failed to load order details' : 'Error';
-    const message = getErrorMessage(error);
-    return <ErrorPage title={title} message={message} />;
+    return <ErrorPage title="Error" message={getErrorMessage(error)} />;
```

Comment:
Note: this will show a more specific error subtitle instead of "Failed to load order details":

<img width="998" alt="image" src="https://github.com/user-attachments/assets/764384f0-080a-4143-a49b-5729d8bbd48a">

Reply:
don't we want to show both? e.g.
```
Failed to load order details:
getaddrinfo ENOTFOUND www.wixapis.com
```

Reply:
It would be a little bit nicer. We could do:

```ts
    try {
        const order = orderId ? await api.getOrder(orderId) : undefined;
        return { order };
    } catch (error) {
        throw json({ message: `Failed to load order. ${getErrorMessage(error)}`}, { status: 500 });
    }
```

Or:
```ts
throw unexpectedErrorResponse('Failed to load order.', error);
```

Because of how unlikely this error is, I'd prefer to keep things simple and just let the original error bubble up.